packages/roosterjs-content-model-core/lib/corePlugin/copyPaste/CopyPastePlugin.ts

```diff
@@ -160,8 +160,8 @@ class CopyPastePlugin implements PluginWithState<CopyPastePluginState> {
                     }
 
                     cleanUpAndRestoreSelection(tempDiv);
-                    this.editor.focus();
                     this.editor.setDOMSelection(selection);
+                    this.editor.focus();
```

Comment:
@juliaroldi would you please help test this change, in Chrome, Firefox and Safari. This is mainly about the cut/copy behavior, and see if the copied content is right, and if focus is back to correct place, for range/image/table selections.

If all work well, you please approve and merge the change.

Thanks.

Reply:
@miku1958 @JiuqingSong @BryanValverdeU this change works fine in Chrome/Firefox/Edge/Safari, the only bug I found was related to cut behavior on tables, but I can also reproduce the bug without this change.